src/Microsoft.AspNetCore.HttpOverrides/ForwardedHeadersMiddleware.cs

```diff
@@ -95,6 +102,37 @@ private static void EnsureOptionNotNullorWhitespace(string value, string propert
             }
         }
 
+        private void PreProcessHosts()
+        {
+            if (_options.AllowedHosts == null || _options.AllowedHosts.Count == 0)
```

Comment:
Nit: Share between the middleware somehow? Add to HttpAbstractions?

Reply:
Not planning on this right now, there are some minor differences.